tensorflow_datasets/testing/dummy_dataset/checksums.txt

```diff
@@ -0,0 +1 @@
+http://yann.lecun.com/exdb/mnist/t10k-labels-idx1-ubyte.gz 4542 f7ae60f92e00ec6debd23a6088c31dbd2371eca3ffa0defaefb259924204aec6
```

Comment:
We also want to use a dummy url here (e.g. http://dummy.org/data.txt). Otherwise, the tests might still pass as url is registered in checksums.get_all_url_infos()